mods/cnc/rules/infantry.yaml

```diff
@@ -161,7 +161,7 @@ E6:
 	Buildable:
 		BuildPaletteOrder: 30
 		Queue: Infantry.GDI, Infantry.Nod
-		Description: Damages and captures enemy structures.\n  Unarmed
+		Description: Damages and captures enemy structures.\n Repairs tank wreckages\n  Unarmed
```

Comment:
Can you please change this to "Repairs destroyed vehicles"?

We have husks for more than just tanks, and refer to them as "Destroyed" rather than wreckages.